presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraRecordSink.java

```diff
@@ -131,7 +147,16 @@ public void appendDouble(double value)
     @Override
     public void appendString(byte[] value)
     {
-        append(new String(value, UTF_8));
+        Type columnType = columnTypes.get(field);
+        if (VARBINARY.equals(columnType)) {
+            append(ByteBuffer.wrap(value));
```

Comment:
`.bind()` method requires all the parameters to be a proper type of an actual column  type